### ISSUE TITLE
Thunkable X Feature Custom Splashscreen

### ISSUE THREAD

**Issue by eoinparkinson** [thunkable/thunkable-issues#175]
Please add the ability to make a custom full screen splash screen / loading screen. The app icon in the centre is boring. A full screen picture support would be brilliant! Please add this!!!!

**Comment by ruymello** [comment]
That would be great!

**Comment by LCR25** [comment]
TOTALLY AGREE, PLEASE, DO SOMETHING!

**Comment by 6118Tool** [comment]
I became a PRO Thunker a few days ago.
I was startled when I saw the Splash Screen with the APP icon.
The design was not cool. And it disagrees with the design of the APP.
We need to customize the Splash Screen.
It will be a good tool to encourage new PRO members.

**Comment by josmas** [comment]
thanks for the reports, all. I am moving this to the internal todo list. Not certain of what we can do and how much flexibility we can offer at the moment, but we'll look into it.

**Comment by 6118Tool** [comment]
Hi Jos,
Thank you very much for your prompt reply.
Just the possibility of disabling the Splash Screen would be a great advance until a definitive solution.

**Comment by eoinparkinson** [comment]
Another addition (even just temporary) would be to change the overall background colour of the splash-screen, so our icons would look cleaner. This might be a quicker addition and definitely better than an all white background. Maybe just add a colour picker in the app properties tab. 

Again only another potential addition, I would like to see some sort of customisation in the near future. 

Thanks for all your hard work towards the platform so far.

**Comment by AmigoDeluxe** [comment]
I agree with @eoinparkinson. It would be nice to be able to control the background color of the splash screen and if possible, the resize mode. I believe that in the app.json file those should be under (example):

    "splash": {
      "resizeMode": "contain",
      "backgroundColor": "#971619"

**Comment by ShtokyD** [comment]
Any progress on this feature yet??
